src/software/amazon/ion/impl/lite/IonStructLite.java

```diff
@@ -401,7 +401,7 @@ public IonValue get(String fieldName)
         IonValue field;
 
         if (field_idx < 0) {
-            if(hasNullFieldName) throw new UnknownSymbolException("Because a field exists with an unknown name we cannot say for certain whether the requested field exists or not.");
+            if(hasNullFieldName) throw new UnknownSymbolException("Unable to resolve fieldname due to null fields.");
```

Comment:
'Null fields' is what's happening within the implementation, but from an Ion perspective isn't the issue. I don't know if this is any less wordy than the original, but here's my stab at it:

"Unable to determine whether the field exists because the struct contains field names with unknown text."